server/models/price.py

```diff
@@ -0,0 +1,31 @@
+from database import db
+import datetime
+
+from .product import Product
+from sqlalchemy.orm import relationship
+from sqlalchemy import ForeignKey
+
+class Price(db.Model):
+
+    __tablename__ = 'prices'
+
+    product = relationship('Product')
+
+    primary_id = db.Column(db.Integer, primary_key=True, unique=True, autoincrement=True)
+    product_id = db.Column(ForeignKey('product.id'), nullable=False)
+    price = db.Column(db.Integer, nullable=False)
```

Comment:
I've removed the `nullable=False` for my products sections because I realized sometimes products may not have a price